stylesheets/main.css

```diff
@@ -58,8 +65,41 @@ form {
     height: 100%;
 }
 
+.map-border {
+    border: 2px solid rgb(202, 200, 200);
+    border-radius: 3px;
+}
+
 .footer {
     display: flex;
     width: 100%;
     height: 5%;
+}
+
+.red-border {
+    border-color: red;
+}
+
+.invalid-input {
+    animation: shake 0.2s linear 4;
```

Comment:
Love it - can we try changing the animation speed to 1.5 seconds?

Reply:
Changed it; looks much better.